src/histogram.js

```diff
@@ -259,9 +266,14 @@ angular.module('grafana.directives').directive('grafanaHistogram', function($roo
           }
         };
 
+        var scopedVars = ctrl.row.scopedVars;
```

Comment:
Doesn't work with panel repeat. (only support row repeat.)
Not sure, but use ctrl.panel.scopedVars seems work with both case.